homeassistant/components/shelly/binary_sensor.py

```diff
@@ -48,13 +53,31 @@
     ),
 }
 
+REST_SENSORS = {
+    ("cloud"): RestAttributeDescription(
```

Comment:
This sensor should be disabled by default. Cloud is not important as we care only about local control.